### PR TITLE
update folder name `contrastive_learning` to match file import paths

### PR DESCRIPTION
**Pull request by rajathpatel23**
- I have updated the folder name to match the file import paths

### CODE REVIEW

**Comment by abrinkmann** [comment]
Hi @rajathpatel23, Can you shortly explain why you want to named the folder "contrastive learning"?

**Comment by rajathpatel23** [comment]
@abrinkmann , yes while I was going through the code I have found path required for pretraining and cross encoder model classses do match current directory naming structure for examples
https://github.com/wbsg-uni-mannheim/SC-Block/blob/main/src/finetuning/open_book/contrastive_pretraining/src/contrastive/run_finetune_cross_encoder.py#L6
https://github.com/wbsg-uni-mannheim/SC-Block/blob/main/src/finetuning/open_book/contrastive_pretraining/src/contrastive/run_pretraining_simclr_deepmatcher.py#L34
which is currently not the case and the MR fixes that. I have also found couple of other bugs which I will try to report and push an MR in for.

**Comment by abrinkmann** [comment]
Great, thank you for the pointers @rajathpatel23. Actually, I would not rename the folder but instead update the imports because the folder contains pretraining scripts for training with supervised contrastive loss, barlow twins and simclr. Barlow Twins and SimCLR are unsupervised methods.

**Comment by rajathpatel23** [comment]
@abrinkmann , that's correct, I did actually update the import paths, on my end earlier, but before creating the PR thought I will try to make minimal changes so I just updated the directory, but I can update the import paths which is much better semantically and  ping back shortly

**Comment by rajathpatel23** [comment]
@abrinkmann  I have updated the commit to the path to match the import paths in the files.

**Comment by rajathpatel23** [comment]
Summary of Changes
- Updated import paths to match the required files
- fix README to show the image correctly

**Comment by abrinkmann** [comment]
@rajathpatel23 Great, thanks for the fixes. I'll merge your changes into the main branch.
